src/parseFunctions.js

```diff
@@ -0,0 +1,23 @@
+import {keyTypes} from './constants'
+
+export const parseQueryToJson=(queryString)=>{
+    const values=decodeURIComponent(queryString).replace('?CMCD=','').split(',');
```

Comment:
Is there any possibility that the URL has some other parameter? 

In that case if I have a URL of the type (I added a ramdomParam): https://urlexmaple/movie.m4v?ramdomParam=test&CMCD=br%3D3200%2Cbs%2Cd%3D4004%2Cmtp%3D25400%2Cot%3Dv%2Crtp

I got this url parsed: https://dash.akamaized.net/akamai/bbb_30fps/bbb_30fps_3840x2160_12000k/bbb_30fps_3840x2160_12000k_0.m4v?ramdomParam': 'test&CMCD'

Reply:
That's a good point! In the issue I commented that this was only meant for the query argument alone, not the complete url. However, I agree with you, that it should also work for the full url. I'll make the changes